### ISSUE TITLE
How to calculate the  connectivity C?

### ISSUE THREAD

**Issue by Monologuethl** [conscienceli/IterNet#2]
I didn't  found in your code,would you please relase the code ?

**Comment by conscienceli** [comment]
yeah, not in the repo currently. Because that code is too ugly. I think I should polish that part a lot before I can release it. But now, I am struggling with several deadlines. So maybe later.
